api/MessagePort.json

```diff
@@ -6,7 +6,7 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/web-messaging.html#message-ports",
         "support": {
           "chrome": {
-            "version_added": "4"
+            "version_added": "1"
```

Comment:
Should the "message_event" and "worker_support" entries also be 1? I'm pretty sure at least the event should be.

Reply:
Updated!  Worker support in general was added in Chrome 4, so that should remain as "4".